src/Tipsy.js

```diff
@@ -203,8 +221,13 @@ Tipsy.propTypes = {
    *
    * You may pass multiple triggers; separate them with a space. Pass an string with value "manual" to manually trigger the tooltip.
    */
-  trigger: PropTypes.string
+  trigger: PropTypes.string,
 
+  /**
+   * Customizable node for rendering the tipsy.
+   *
+   */
+  container: React.PropTypes.node,
```

Comment:
Is node correct? Getting proptype errors in our internal implementation:
https://github.com/mavenlink/mavenlink/pull/8002/commits/09c030793b5c482db473d54042882e4c40dd10bb#diff-5ff172b8e3e60757abce818732618647R20